File: Algorithm_class/Codes/Python/kvadrat_tenlik.py
a, b, c = map(float, input().split())
d = b*b - 4*a*c
if d < 0: print("Kok yoxdur")
elif d == 0: print("Tek kok:", -b / (2*a))
else:
    x1 = (-b - d**0.5) / (2*a)
    x2 = (-b + d**0.5) / (2*a)
    print(x1, x2)

File: Algorithm_class/Codes/C++/kvadrat_tenlik.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main() {
    double a, b, c;
    cin >> a >> b >> c;
    double d = b*b - 4*a*c;
    if (d < 0) cout << "Kok yoxdur" << endl;
    else if (d == 0) cout << "Bir kok: " << -b/(2*a) << endl;
    else {
        double x1 = (-b - sqrt(d))/(2*a);
        double x2 = (-b + sqrt(d))/(2*a);
        cout << "Kokler: " << x1 << " " << x2 << endl;
    }
    return 0;
}
